no-computer.md

```diff
@@ -11,41 +11,81 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need met by my idea is the need to play a large range of musical notes which
+can be composed together to make music. I am helping pretty much anyone who would
+want to be able to easily create music and chain notes together to create more complex
+creations. Right now it is difficult to use the entire musical range. Hopefully with
```

Comment:
Even string instruments have an upper range - your hand can only move so far up.

Additionally, different instruments also have different timbres, and this can affect what ranges are their "ideal" range (which also varies depending on the desired sound/effect)